src/snapshot.rs

```diff
@@ -131,11 +131,37 @@ impl Snapshot {
             .collect()
     }
 
-    pub(crate) fn keys_at_ts<'a, R>(&'a self, range: R, ts: u64) -> Vec<&'a [u8]>
+    pub(crate) fn keys_at_ts<'a, R>(
+        &'a self,
+        range: R,
+        ts: u64,
+        limit: Option<usize>,
+    ) -> Vec<&'a [u8]>
     where
         R: RangeBounds<VariableSizeKey> + 'a,
     {
-        self.snap.keys_at_ts(range, ts).collect()
+        let iter = self.snap.keys_at_ts(range, ts);
```

Comment:
Aren't we removing `keys_at_ts` in https://github.com/surrealdb/vart/pull/79?

Reply:
Yup, chucking this

Reply:
Removed! Just rewrote this PR